src/modules/commentPage.test.js

```diff
@@ -0,0 +1,5 @@
+const commentPage = require('./commentPage.js');
+
```

Comment:
The test is not working. I suggest you try it by creating a separate function for the comment counter. Moreover, please add more cases to test the edge cases.